site/en/blog/how-fugu-is-my-browser-and-how-fugu-is-the-web/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: 'layouts/blog-post.njk'
+title: '"How Fugu is my browser?" and "How Fugu is the web?"'
```

Comment:
I'm not a fan of this title, I think it's going to be meaningless to a lot of people. Can you come up with something descriptive?